src/AudioToolbox/AudioFile.cs

```diff
@@ -213,6 +216,45 @@ public struct AudioFileMarker
 		}
 	}
 
+	[Introduced (PlatformName.UIKitForMac, 13,0)]
+	[NoWatch, iOS (13,0), Mac (10,15, onlyOn64 : true), TV (13,0)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AudioPacketRangeByteCountTranslation
+	{
```

Comment:
I believe that the opening curly braces should not be on the new line except for with methods